mantaray/views.py

```diff
@@ -542,6 +547,27 @@ def on_part(self, nick: str, reason: str | None) -> None:
             show_in_gui=self.server_view.should_show_join_leave_message(nick),
         )
 
+    def on_kick(self, channel: str, kicker: str, kicked_nick: str, reason: str | None) -> None:
+        self.userlist.remove_user(kicked_nick)
+        if reason is None:
+            reason = ''
+        if kicker == self.server_view.core.nick:
+           kicker_tag = 'self-nick'
+        else:
+           kicker_tag = 'other-nick'
+        kicked_tag=[]
+        if kicked_nick == self.server_view.core.nick:
+           kicked_tag = 'self-nick'
+        else:
+           kicked_tag = 'other-nick'
+        self.add_message(
+            "*",
+            (kicker, [kicker_tag]),
+            (' has kicked ' , []),
+            (kicked_nick, [kicked_tag]),
+            (" from the channel." + f"     (Reason: {reason})", []))
```

Comment:
- This line can be rewritten with just one string, no need to `+` two strings together
- Why so many spaces before reason?

Reply:
Spaces erased, line rewritten.